modules/salt/files/bin/rename-wiki.py

```diff
@@ -21,10 +25,10 @@ class DbClusterMap(TypedDict):
 
 
 def generate_salt_command(cluster: str, command: str) -> str:
-    return f'salt-ssh -E "{cluster}*" cmd.run "{command}"'
+    return f'salt-ssh -E "{cluster}" cmd.run "{command}"'
```

Comment:
IIRC we don't need -E if we aren't using * but isn't * required if the full hostname is IE db151.wikitide.net? I can't remember if salt uses that or just db151 though